shared/actions/chat/index.js

```diff
@@ -1006,6 +1080,14 @@ function* _exitSearch() {
 function* chatSaga(): SagaGenerator<any, any> {
   yield Saga.safeTakeEvery('app:changedFocus', _changedFocus)
   yield Saga.safeTakeEvery('chat:appendMessages', _sendNotifications)
+  yield Saga.safeTakeEvery('chat:clearMessages', _clearConversationMessages)
+  yield Saga.safeTakeEvery(['chat:appendMessages', 'chat:prependMessages'], _storeMessageToEntity)
+  yield Saga.safeTakeEvery(['chat:appendMessages', 'chat:prependMessages'], _findMessagesToDelete)
+  yield Saga.safeTakeEvery(['chat:appendMessages', 'chat:prependMessages'], _findMessageUpdates)
+  yield Saga.safeTakeEvery(['chat:updateMessage', 'chat:updateTempMessage'], _updateMessageEntity)
```

Comment:
We can get rid of updateTempMessage by moving the fields we want to change to their own entity map of `{MessageKey: state}.` For example, we mostly call this to change messageState (e.g. to `failed`). We could have `entities.messageState: Map<MessageKey: MessageStatus>` and have a better action called `chat:updateMessageState = (msgKey, msgState)`